pythainlp/tokenize/__init__.py

```diff
@@ -139,10 +139,11 @@ def word_tokenize(
         from .pyicu import segment
 
         segments = segment(text)
-    else:  # default, use "newmm" engine
-        from .newmm import segment
-
-        segments = segment(text, custom_dict)
+    else:
+        raise ValueError(
+            f"""\"{engine}\""" not found.
```

Comment:
It seems we have two additional `"` after `\"`.

I purpose we can the statement to
```
raise ValueError(
    f"""\"{engine}\" not found.
It might be a typo; if not, please consult our document."""
)

# output:
ValueError: "XXX" not found.
It might be a typo; if not, please consult our document.
```

Reply:
> 
> 
> It seems we have two additional `"` after `\"`.
> 
> I purpose we can the statement to
> 
> ```
> raise ValueError(
>     f"""Tokenizer \"{engine}\" not found.
> It might be a typo; if not, please consult our document."""
> )
> 
> # output:
> ValueError: Tokenizer "XXX" not found.
> It might be a typo; if not, please consult our document.
> ```

Fixed #368